sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjb.java

```diff
@@ -446,6 +446,14 @@ public EventParticipantReferenceDto getReferenceByUuid(String uuid) {
 		return new EventParticipantReferenceDto(eventParticipant.getUuid());
 	}
 
+	@Override
+	public EventParticipantReferenceDto getReferenceByEventAndPerson(String eventUuid, String personUuid) {
+		Event event = eventService.getByUuid(eventUuid);
+		EventParticipant eventParticipant =
+			event.getEventPersons().stream().filter(ep -> !ep.isDeleted() && ep.getPerson().getUuid().equals(personUuid)).findFirst().orElse(null);
+		return new EventParticipantReferenceDto(eventParticipant.getUuid());
```

Comment:
I suspect this throws a NullPointerException if no event participant is found above because you explicitely set it to null in that case.

Reply:
It does ! It didn't take care of it because it there's the same thing with the getReferenceByUuid right above and without really thinking about it I was telling myself it shouldn't happen (that, and maybe lazyness :tongue:). But it is indeed going to happen (another user already deleted the EP in the meantime => :boom:).
I changed that and added a proper error handling.